test/getCommands.spec.js

```diff
@@ -1,50 +1,46 @@
 const path = require('path');
-const Module = require('module');
 const expect = require('chai').expect;
 const getCommands = require('../src/getCommands');
-const findPlugins = require('../src/findPlugins');
-const mockRequire = require('mock-require');
-const mockFs = require('mock-fs');
+const mock = require('mock-require');
 
-const commands = getCommands();
-const filesPath = path.join(__dirname, 'fixtures', 'files');
-const testPlugin = findPlugins(filesPath);
-const testPluginPath = path.join(__dirname, '..', 'node_modules', 'rnpm-plugin-test');
-const singlePlugin = require(path.join(filesPath, 'plugin'));
-const multiPlugin = require(path.join(filesPath, 'plugins'));
+const commands = require('./fixtures/commands');
+const plugins = require('./fixtures/plugins');
 
-describe('getCommands', () => {
+const pluginFolder = path.join(process.cwd(), 'node_modules', 'rnpm-plugin-link');
 
-  before(() => {
-    mockRequire(path.join(__dirname, '..'), {});
-    mockRequire(
-      path.join(__dirname, '..', 'package.json'),
-      require(path.join(filesPath, 'package'))
-    );
-  });
+describe('getCommands', () => {
 
   it('list of the commands should be an array', () => {
-    expect(commands).to.be.an('array');
+    expect(getCommands()).to.be.an('array');
   });
 
   it('should contain some pre-defined commands by default', () => {
-    expect(commands).to.be.not.empty;
+    expect(getCommands()).to.be.not.empty;
   });
 
   it('should return a single command (plugin export one command)', () => {
-    mockRequire(testPluginPath, singlePlugin);
+    mock(pluginFolder, commands.single);
```

Comment:
Since our package.json contains rnpm-plugin-link, we mock it (pluginFolder) to test out various cases.